cmd/virt-launcher/virt-launcher.go

```diff
@@ -29,6 +29,8 @@ import (
 	"syscall"
 	"time"
 
+	"kubevirt.io/kubevirt/pkg/downwardmetrics"
+
```

Comment:
Move this to the large block at the end of the imports?

Reply:
Done.